### ISSUE TITLE
Dont have proximity notification active by default

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#7756]
Users feedback regarding our beta version 2019.07.25-RC on social media:

> I noticed yesterday when using the compass to navigate to a cache, it's now "beeping". I get a beep at a certain distance that switches to three beeps when I get closer? 

> Seit heute gibt mein Handy seltsame Geräusche von sich, wenn ich mich einem Cache nähere und den Kompass an habe. Kann ich das irgendwo abschalten?

Obviously the new proximity notification seems to be active by default.
IMHO we should change this to not being active by default. While this would be nice to make users aware of this feature, I also understand, that it might be annoying if your app behaves unexpected.

**Comment by moving-bits** [comment]
> Obviously the new proximity notification seems to be active by default.

Hm, the settings both default to `false` in `Settings.java`, must be a different reason:
```
public static boolean isGeneralProximityNotificationActive() {
   return getBoolean(R.string.pref_proximityNotificationGeneral, false);
}

public static boolean isSpecificProximityNotificationActive() {
    return getBoolean(R.string.pref_proximityNotificationSpecific, false);
}
```

I'll check further.

**Comment by moving-bits** [comment]
Check for `Settings.isSpecificProximityNotificationActive()` was missing in CompassActivity - created PR #7759 for it.

**Comment by Lineflyer** [comment]
Thanks.
Thats what beta testing is good for. I checked the behaviour after installation but missed to check the compass.

**Comment by moving-bits** [comment]
Closed with #7759

**Comment by Lineflyer** [comment]
Auto close did not yet catch this as the change is not yet on master.
Let me test the auto close feature when I merge branches now.

**Comment by Lineflyer** [comment]
OK, it did not close because there is no "fix #7756" in the commit message. :)

**Comment by moving-bits** [comment]
I thought this would work for both the commit message and the comment title/info you enter on github when you create the PR.
Adding fix #xxxx to the commit message has the side effect that it creates a reference comment in the referenced issue, which is useful when being the first time, but it gets created each time anew when you add something to your commit, potentially cluttering the issue.
